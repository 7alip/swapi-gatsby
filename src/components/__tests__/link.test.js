import React from "react"
import renderer from "react-test-renderer"

import Link from "../link"

describe("Counter", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Link to="/test">Test</Link>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
