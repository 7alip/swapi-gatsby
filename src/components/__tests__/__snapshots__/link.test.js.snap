// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter snapshot renders 1`] = `
.emotion-0 {
  -webkit-transition: all 0.15s ease-out;
  transition: all 0.15s ease-out;
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  outline: none;
  color: blue.400;
}

.emotion-0:hover {
  color: blue.600;
}

.emotion-0:focus {
  box-shadow: none;
}

.emotion-0:disabled,
.emotion-0:disabled:focus,
.emotion-0:disabled:hover,
.emotion-0[aria-disabled=true],
.emotion-0[aria-disabled=true]:focus,
.emotion-0[aria-disabled=true]:hover {
  opacity: 0.4;
  cursor: not-allowed;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<a
  className="emotion-0"
  href="/test"
>
  Test
</a>
`;
